common/views/components/BaseHeader/BaseHeader.js

```diff
@@ -60,14 +61,10 @@ const BaseHeader = ({
               ${grid({s: 12, m: 10, shiftM: 1, l: 8, shiftL: 2, xl: 8, shiftXL: 2})}
               ${spacing({s: 2}, {padding: ['bottom']})}
             `}>
-              <h1 className={`
-              h1 inline-block no-margin
-              ${Background ? '' : `
-                bg-white
-                ${spacing({ s: 2 }, { padding: ['left', 'right'] })}
-                ${spacing({ s: 1 }, { padding: ['bottom', 'top'] })}
-              `}
-            `}>{title}</h1>
+
+              <BaseHeading
```

Comment:
I'd argue this isn't a `BaseHeading`, but a `HighlightedHeading`, or something to that regard, which is used in the `BaseHeader` (`BaseHeader` as it is the Base of / composed into a `Header`).

Reply:
Yeah, I wondered that. But it's only conditionally 'highlighted'. Have you got a better name than `<MaybeHighlightedHeading>`?

Reply:
Is it maybe up to the template before then to choose?
Then the BaseHeader can take a title of type `HighlightedHeading` of `<h1>`?